extensions/wikia/Helios/tests/UserTest.php

```diff
@@ -17,32 +17,66 @@ public function setUp()
 		parent::setUp();
 	}
 
-	public function testNewFromTokenNoAuthorizationHeader()
+	public function testGetAccessTokenFromCookie()
+	{
+		$token = 'qi8H8R7OM4xMUNMPuRAZxlY';
+
+		$this->webRequestMock->expects( $this->once() )
+			->method( 'getCookie' )
+			->willReturn( $token );
+
+		$this->assertEquals( User::getAccessToken( $this->webRequestMock ), $token );
+	}
+
+	public function testGetAccessTokenFromAuthorizationHeader()
+	{
+		$token = 'qi8H8R7OM4xMUNMPuRAZxlY';
+
+		$this->webRequestMock->expects( $this->once() )
+			->method( 'getCookie' )
+			->willReturn( null );
+
+		$this->webRequestMock->expects( $this->once() )
+			->method( 'getHeader' )
+			->with( 'AUTHORIZATION' )
+			->willReturn( "Bearer $token" );
+
+		$this->assertEquals( User::getAccessToken( $this->webRequestMock ), $token );
+	}
```

Comment:
Would it make sense to test the case when both cookie and header are set, but with a different token (i.e. test that the cookie is checked before HTTP header)?